### ISSUE TITLE
Website ("Publish your game" site)

### ISSUE THREAD

**Issue by dataGuyJosh** [thommoboy/There-are-no-brakes#183]
Best possibility: a custom-built site on your own domain.

Must Have:
- You have full control over its contents
- Open to public: no login/signup required to see your content
- Pitches your game with excitement and professionalism
- Easy, prominent link to download your setup / play directly
- Easy way to contact you
- Publically visible means to provide and read feedback and comments
- Tracks number of downloads/uses of your product
- Download URL for game

Nice to have:
- URL not too arcane
- star system or other ratings
- news feeds for fans to track you **(our dev blog probably covers this)**
- pages and articles about you **(make a "the team" or some equivalent section?)**, how the game was made, what you're doing next **(make a plan/roadmap section?)**

"Various fun froufrou"
- dev diaries
- wacky playthrough videos
- desktop backgrounds
- t-shirts with game/company logo

**Comment by NAT0P0TAT0** [comment]
basically have everything in the 'must have' and half the 'nice to have' stuff, mostly just waiting on the domain verification thing
